doc/modules/changes/20210713_jdannberg

```diff
@@ -0,0 +1,6 @@
+New: There is now a rising velocity output variable in the depth 
+average postprocessor.
+Fixed: The sinking velocity depth average postprocessor now computes 
+the actual sinking velocity (and not the rising velocity).  
+<br>
```

Comment:
I don't see this change...

Reply:
I changed this in #4186 and @gassmoeller merged it and told me to add the changelog entry in this follow-up.

Reply:
Or do you mean the Fixed part? That's changing the std::min to std::max in the lateral_averaging.cc.

Reply:
Nevermind, I didn't check the other pr